src/preprocess/base.py

```diff
@@ -159,10 +159,10 @@ def regrid(
             output_dict[var] = regridder(ds[var])
         ds = xr.Dataset(output_dict)
 
-        print(
-            f"Regridded from {(regridder.Ny_in, regridder.Nx_in)} "
-            f"to {(regridder.Ny_out, regridder.Nx_out)}"
-        )
+        # print(
+        #     f"Regridded from {(regridder.Ny_in, regridder.Nx_in)} "
```

Comment:
is this remaining commented out?

Reply:
oops... fixed! thanks for catching this. Fixed!